string-manipulation-1/src/capitalize.js

```diff
@@ -0,0 +1,10 @@
+/* exported capitalize */
+function capitalize(word) {
+  var firstLetter = word.slice(0, 1);
+  var firstLetterUpperCased = firstLetter.toUpperCase();
+
+  var rest = word.slice(1, word.length);
```

Comment:
You can just do `word.slice(1)`. If given only one argument, `slice` will cut from the start point to the end of the string.